src/main/resources/org/jenkinsci/test/acceptance/docker/fixtures/JavaContainer/Dockerfile

```diff
@@ -18,5 +18,8 @@ ENTRYPOINT ["/usr/sbin/sshd", "-D", "-e"]
 
 # JDK is from Universe
 RUN echo deb http://archive.ubuntu.com/ubuntu precise universe >> /etc/apt/sources.list
+RUN apt-get install --no-install-recommends -y software-properties-common
+RUN add-apt-repository ppa:openjdk-r/ppa
 RUN apt-get update
+RUN apt-get install --no-install-recommends -y openjdk-8-jdk
```

Comment:
it is better to run all installations in a single line as it produces less intermediary images.